dev/Interop/StoragePickers/PickerLocalization.cpp

```diff
@@ -9,14 +9,47 @@
 #include "PickerLocalization.h"
 #include <winrt\base.h>
 #include <winrt\Microsoft.Windows.ApplicationModel.Resources.h>
-#include <iostream>
+#include <array>
+
+
+const winrt::hstring priFileName = L"Microsoft.WindowsAppRuntime.pri";
+winrt::hstring GetPriFilePath()
+{
+    HMODULE hModule = nullptr;
+    // use GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT, the module handle does not need to be released
+    // GetPriFilePath should only be called once to initialize the static priFilePath variable, thus thread safe
+    if (GetModuleHandleExW(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS | GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT,
+        reinterpret_cast<LPCWSTR>(&GetPriFilePath), &hModule))
+    {
+        std::array<wchar_t, MAX_PATH + 1> buffer;
+        auto pathLength = GetModuleFileName(hModule, buffer.data(), MAX_PATH + 1);
+        if (pathLength > 0)
+        {
+            auto dllPathString = std::wstring(buffer.data(), pathLength);
+            std::filesystem::path dllPath(dllPathString);
+            if (!std::filesystem::is_regular_file(dllPath))
+            {
+                return priFileName;
+            }
```

Comment:
Use WIL, not raw Win32 APIs

Replace lines 15-43 using a variant of `DeploymentManager::GenerateDeploymentAgentPath()`

```
winrt::hstring GetPriFilePath()
{
    wil::unique_hmodule module;
    THROW_IF_WIN32_BOOL_FALSE(GetModuleHandleEx(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS, reinterpret_cast<PCWSTR>(&GetPriFilePath), &module));

    std::filesystem::path modulePath{ wil::GetModuleFileNameW<std::wstring>(module.get()) };
    const auto priFilename{ modulePath.parent_path() / priFileName };
    if (std::filesystem::exists(priFilePath))
    {
        return winrt::hstring{ priFilename.c_str() };
    }

    PCWSTR c_WindowsAppRuntimePriFilename{ L"Microsoft.WindowsAppRuntime.pri" };
    return winrt::hstring{ c_WindowsAppRuntimePriFilename };
```